solutions/quickstart/variables.tf

```diff
@@ -277,6 +290,24 @@ variable "override_json_string" {
                            "port_min": 22
                          }
               },
+	      {
+                  "direction": "inbound",
+                  "name": "allow-all-inbound-1",
+                  "source": "0.0.0.0/0",
+                  "tcp": {
+                           "port_max": 992,
+                           "port_min": 992
```

Comment:
Why this change?   Its not related to your PR.

Reply:
I created this new branch out of standard_quickstart and it was throwing error conflict. 
Reverted back the changes and copied the override_json_string values from init_quickstart branch.